main/src/main/java/org/mobilitydata/gtfsvalidator/validator/StopTimeTravelSpeedValidator.java

```diff
@@ -72,25 +73,51 @@
 
   @Override
   public void validate(NoticeContainer noticeContainer) {
-    final ListMultimap<Long, List<GtfsStopTime>> tripsByHash = ArrayListMultimap.create();
+    final ListMultimap<Long, TripAndStopTimes> tripsByHash = ArrayListMultimap.create();
     for (List<GtfsStopTime> stopTimes : Multimaps.asMap(stopTimeTable.byTripIdMap()).values()) {
-      tripsByHash.put(
-          tripFprint(tripTable.byTripId(stopTimes.get(0).tripId()), stopTimes), stopTimes);
+      tripTable
+          .byTripId(stopTimes.get(0).tripId())
+          .ifPresent(
```

Comment:
I think this might be simplified a bit. First `tripFprint()` method should be moved into TripAndStopTimes class. Then we can have something like

tripTable
          .byTripId(stopTimes.get(0).tripId())
          .map(trip -> new TripAndStopTimes(trip, stopTimes))
          .ifPresent(tripAndStopTimes -> tripsByHash.put(tripAndStopTimes.tripFprint(), tripAndStopTimes));